packages/core/src/actions/smartAccount/buildUserOperationFromTxs.ts

```diff
@@ -77,7 +78,15 @@ export const buildUserOperationFromTxs: <
   };
   filterUndefined(_overrides);
 
+  const uoStruct = await buildUserOperation(client, {
+    uo: batch,
+    overrides: _overrides,
+    account: account as SmartContractAccount,
+  });
```

Comment:
after reviewing, I think we should leave this method as is.

I think a todo for v4 would be:
1. rename this to `convertTxToUserOperation`
2. add `buildUserOperationFromTx` which calls `buildUserOperation` as you have it here

For now, if someone wants to actually `buildUserOperationFromTx` it would look like:

```ts
const { batch, overrides } = client.buildUserOperationFromTx(...);
const result = client.buildUserOperation({ uo: batch, overrides });
```

Reply:
ah wait nevermind... it's just the `Txs` one that wasn't calling `buildUo` never mind